addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -46,6 +50,7 @@ def send_mail_test(self):
         else:
             full_body = mailing._prepend_preview(mailing.body_html, mailing.preview)
             subject = mailing.subject
+        subject = f'[TEST] {subject}'
```

Comment:
Not sure for this one as it's used just for testing mailing, but should we translate it ?